### PR TITLE
feat(subcommands): type improvements & fix build

### PR DESCRIPTION
**Pull request by Favna**

### CODE REVIEW

File: packages/subcommands/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+_With TypeScript:_
+
+```typescript
+import { SubCommandPluginCommand } from '@sapphire/plugin-subcommands';
+import { ApplyOptions } from '@sapphire/decorators';
+import type { Args } from '@sapphire/framework';
+import type { Message } from 'discord.js';
+
+// Using ApplyOptions decorator makes it easy to configure
+@ApplyOptions<SkyraCommand.Options>({
```
Comment (vladfrangu) [inline packages/subcommands/README.md:49]:
```suggestion
@ApplyOptions<SubCommandPluginCommand.Options>({
```

**Review by vladfrangu — DISMISSED** [review]
The names are monkaw, fine otherwise

**Review by Stitch07 — APPROVED** [review]

File: packages/subcommands/README.md (hunk 1 above)
Comment (Favna) [reply packages/subcommands/README.md]: done

**Review by vladfrangu — APPROVED** [review]

**Review by kyranet — APPROVED** [review]
As long as I don't have to read those types breaking into different lines myself... LGTM

**Pull request merged**